futures-util/tests/foo.rs

```diff
@@ -0,0 +1,18 @@
+#![feature(async_await)]
+use futures::future::{FutureExt, TryFutureExt};
+use futures::compat::Executor01CompatExt;
+use tokio_threadpool::ThreadPool;
+
+#[test]
+fn foo () {
```

Comment:
Could do with a better name for this file and test

Reply:
I never meant to commit this file...